docs/sources/articles/remote_api_authorization.md

```diff
@@ -0,0 +1,63 @@
+# Docker's remote API authorization with macaroons
+
+Docker uses [Macaroons](http://research.google.com/pubs/pub41892.html) for authorize requests to the remote API. This article explains how to grant privileges for a remote client to interact with the API.
+
+## Initializing the credentials in the server
+
+The first command to run is `docker id --init`. This will create the root macaroon in the server and it will be shared with the client.
```

Comment:
Definitely, I think this proposal still needs a more detailed part explaining how all this works from the client perspective.